### ISSUE TITLE
Abadoned? Anybody able to work with me to fix this and get it back up?

### ISSUE THREAD

**Issue by amrithHN** [eclipse-color-theme/eclipse-color-theme#292]
I want this to be resurrected who are willing to work one this?

**Comment by PyvesB** [comment]
See #273 :)

**Comment by hapeeeeee** [comment]
I  willing to work for this. Would you like to cooperate with me?

**Comment by amrithHN** [comment]
> I willing to work for this. Would you like to cooperate with me?

@hapeeeeee yes. I didnt notice your reply until now. should we fork this repo ?

**Comment by hapeeeeee** [comment]
> > I willing to work for this. Would you like to cooperate with me?
> 
> [@hapeeeeee](https://github.com/hapeeeeee) yes. I didnt notice your reply until now. should we fork this repo ?

Actually, I forked this project six months ago, but I haven't started working on it yet. I'm a beginner in Java/Eclipse/IDE development, and if you're willing, we can start rebuilding this project together.

**Comment by Philipp0205** [comment]
@hapeeeeee and @amrithHN I would also offer my help, did anybody already start working on it or do you need somebody to drive it?

**Comment by ahatem** [comment]
Hey @amrithHN, @PyvesB, @hapeeeeee, and @Philipp0205!

Just saw you were all looking to build a new theme plugin. I actually built one along with the theme website I announced back in #290 ([eclipse-color-themes.vercel.app](https://eclipse-color-themes.vercel.app/)).

It's already submitted to the Eclipse Marketplace and is waiting for review. In the meantime, if you're still interested, you can install it directly from the GitHub update site to try it out.

**Update Site URL:** `https://ahatem.github.io/eclipse-themes-plugin/`

**Source code:** [https://github.com/ahatem/eclipse-themes-plugin](https://github.com/ahatem/eclipse-themes-plugin)

Hope this saves you all some effort! I'd love to hear what you think, and of course, contributions are welcome.

Cheers!

**Comment by Philipp0205** [comment]
@ahatem Amazing!! I am looking forward to taking a look at your plugin tomorrow. I forked the project at one point but just tinkered a bit around; it was far from a finished plugin. I am glad you did the work 😉
